docs(nuxt): Add Nuxt docs for manual setup (#10981)
getsentry/sentry-docs: platform-includes/getting-started-sourcemaps/javascript.nuxt.mdx

## Add Readable Stack Traces to Errors

The Sentry Nuxt Module uses the [Sentry Vite Plugin](https://www.npmjs.com/package/@sentry/vite-plugin) to upload source maps for both server and client builds.
This means that when you run a production build (`nuxt build`), source maps will be generated and uploaded to Sentry, so that you get readable stack traces in your Sentry issues.

> To upload source maps, you need to specify your Sentry auth token as well as your org and project slugs. You can specify those in the `sourceMapsUploadOptions` option
> inside the `sentry` options of your `nuxt.config.ts`.

Reviewer: l: l: I'd use "you can" formulations sparingly. Our general "voice"/mentality is that we are opinionated and tell people what they should do to get the SDK set up.

```suggestion
To upload source maps, specify your Sentry auth token as well as your org and project slugs. Set them in the `sourceMapsUploadOptions` option
inside the `sentry` options of your `nuxt.config.ts`.
```

(I'm aware that we're probably having similar wording in countless places but I think we should try to be more concise and straight forward when we have the chance)

(Also if someone from docs teams sees this differently, please feel free to roast me :D)

<Alert level="info">
  The module options inside `sentry` are only affecting the **build-time** of the SDK.
</Alert>

<OrgAuthTokenNote />

> ```javascript {filename:nuxt.config.ts} {3-10}
> export default defineNuxtConfig({
>   modules: ["@sentry/nuxt/module"],
>   sentry: {
>     debug: true,
>     sourceMapsUploadOptions: {
>       org: "___ORG_SLUG___",
>       project: "___PROJECT_SLUG___",
>       authToken: "___ORG_AUTH_TOKEN___"
>     }
>   }
> });
> ```

Reviewer: l: I guess `debug` was a leftover here?

```suggestion
```javascript {filename:nuxt.config.ts} {3-9}
export default defineNuxtConfig({
  modules: ["@sentry/nuxt/module"],
  sentry: {
    sourceMapsUploadOptions: {
      org: "___ORG_SLUG___",
      project: "___PROJECT_SLUG___",
      authToken: "___ORG_AUTH_TOKEN___"
    }
  }
});
```
```
